mrmustard/lab_dev/wires.py

```diff
@@ -27,48 +27,48 @@
 
 class Wires:
     r"""
-    A class with wire functionality for tensor network applications.
+     A class with wire functionality for tensor network applications.
```

Comment:
Are these extra white spaces intentional?

Reply:
ugh, no idea why my black inserted them

Reply:
should be fixed now